### DISCUSSION TITLE
Traceback Error - What do I do?

### DISCUSSION THREAD

**Discussion by AtlasRover** [imartinez/privateGPT#905] [Q&A]
Hello,

While running "ingest.py", after successful pip3 installation from requirements, the following traceback error is arising -

"PS F:\New folder> python ingest.py
Traceback (most recent call last):
File "F:\New folder\ingest.py", line 27, in
from constants import CHROMA_SETTINGS
File "F:\New folder\constants.py", line 11, in
CHROMA_SETTINGS = Settings(
^^^^^^^^^
File "pydantic\env_settings.py", line 40, in pydantic.env_settings.BaseSettings.init
File "pydantic\main.py", line 341, in pydantic.main.BaseModel.init
pydantic.error_wrappers.ValidationError: 1 validation error for Settings
persist_directory
none is not an allowed value (type=type_error.none.not_allowed)
"

Could you help me in figuring out what the issue is? Thank you so much in advance.

**Comment by Clydesdale404** [comment]
Got the same issue Myself - Let me know if you find the solution!

**Comment by Clydesdale404** [comment]
I worked around it in the short term by modifying the injest.py file directly and setting the directly in ' '   I,E,  'db'
